### PR TITLE
Use 'mobile' for Android

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by LeoMcA** [comment]
:thinking: https://github.com/mozilla/kitsune/pull/4603#pullrequestreview-534246669

File: kitsune/questions/tests/test_utils.py [hunk 1]
```diff
@@ -107,10 +108,6 @@ class GetMobileProductFromUATests(TestCase):
                 "Mozilla/5.0 (iPhone; CPU iPhone OS 12_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) FxiOS/7.0.4 Mobile/16B91 Safari/605.1.15",  # noqa: E501
                 "ios",
             ),
-            (
-                "Mozilla/5.0 (Android 10; Mobile; rv:76.0) Gecko/76.0 Firefox/76.0",
-                "firefox-preview",
```
Comment (LeoMcA) [inline kitsune/questions/tests/test_utils.py:112]: I'd keep this test, but change our desired string to "mobile".

File: kitsune/questions/utils.py [hunk 2]
```diff
@@ -45,12 +44,4 @@ def get_mobile_product_from_ua(user_agent):
...
 
     # android
-    try:
-        # We are using firefox instead of Firefox as lower() has been applied to the UA
-        mobile_client = re.search(r"firefox/(?P<version>\d+)\.\d+", ua).groupdict()
-    except AttributeError:
-        return None
-    else:
-        if int(mobile_client["version"]) >= 69:
-            return "firefox-preview"
```
Comment (LeoMcA) [inline kitsune/questions/utils.py:55]: I think it's only this block we want to remove, we need try/except above so non-firefox browsers on android return `None` - I think this is why the tests are failing.

**Review by LeoMcA — APPROVED** [review]
r+wc

**Comment by akatsoulas** [comment]
> thinking [#4603 (review)](https://github.com/mozilla/kitsune/pull/4603#pullrequestreview-534246669)

Hehe, I completely forgot about that

**Pull request merged**